libs/common/src/types/dtos/transaction.dto.ts

```diff
@@ -2,11 +2,14 @@
 import { BlockHash, Hash } from '@polkadot/types/interfaces';
 import { PublishHandleRequest } from './handles.dto';
 import { PublishSIWFSignupRequest } from './wallet.login.request.dto';
+import { SIWFSignupRequest } from './wallet.login.response.dto';
+import { PublishKeysRequest } from './keys.dto';
 
-export type TransactionData<RequestType = PublishHandleRequest | PublishSIWFSignupRequest> = RequestType & {
-  providerId: number;
-  referenceId: string;
-};
+export type TransactionData<RequestType = PublishHandleRequest | SIWFSignupRequest | PublishKeysRequest> =
```

Comment:
I need to check this type:
```suggestion
export type TransactionData<RequestType = PublishHandleRequest | PublishSIWFSignupRequest | PublishKeysRequest> =
```